evalml/tests/model_understanding_tests/test_permutation_importance.py

```diff
@@ -441,7 +477,7 @@ def test_get_permutation_importance_binary(
         parameters={"Logistic Regression Classifier": {"n_jobs": 1}}, random_seed=42
     )
     pipeline.fit(X, y)
-    for objective in binary_core_objectives:
+    for objective in ["Log Loss Binary", "AUC"]:
```

Comment:
Running fewer objectives and columns

Reply:
Curious, would it be better to replace `binary_core_objectives` with something like a `binary_test_objectives`, then we could just reduce the load on all the tests at once.  Additionally, is the fixture used after you pull it out?  Seems kinda weird that lint didn't pick it up...

Reply:
Word I'd be down for changing the fixture! I just wanted to change the slowest tests first. What are good candidates for `binary_test_objectives`?

Reply:
I think F1, Log Loss, and AUC could be good choices

Reply:
Done!